phpunit.xml.dist

```diff
@@ -19,9 +17,10 @@
     </testsuites>
 
     <filter>
-        <whitelist addUncoveredFilesFromWhitelist="false">
+        <whitelist addUncoveredFilesFromWhitelist="true">
             <directory suffix=".php">src</directory>
             <exclude>
+                <directory suffix=".php">tests</directory>
                 <directory suffix=".php">vendor</directory>
```

Comment:
this exclude rules are useless, as ``tests`` and ``vendor`` are not inside ``src``